New activity log UI

heretge opened the pull request:
https://www.pivotaltracker.com/story/show/165692434

`Audited::Audit` is now independent of database connection and serves as an entity in our codebase.

krasnoukhov commented:
@NoahFisher Ideally I'd like to switch over to something more appropriate to our use case. Here's a story: https://www.pivotaltracker.com/story/show/168167606

This is meant as a band-aid to keep the current codebase running

krasnoukhov commented:
I'll let @heretge elaborate on reasoning. But I think it's fine to use this bandaid for now to not complicate the other PRs even further and use something else in place of this in a later PR (for which we have a story in the epic).

krasnoukhov commented on lib/audited/audit.rb:
```diff
@@ -1,190 +1,160 @@
+  before_create :set_version_number, :set_audit_user, :set_request_uuid, :set_remote_address
+
+  # All audits made during the block called will be recorded as made
+  # by +user+. This method is hopefully threadsafe, making it ideal
```
This is the original comment from the gem 🤷‍♂

krasnoukhov commented on lib/audited/audit.rb:
```diff
@@ -1,190 +1,160 @@
-      end
-    end
+  def audit_job_call(_attributes)
+    # no-op
```
I guess it so tests here in the gem work

heretge commented on lib/audited/audit.rb:
```diff
@@ -1,190 +1,160 @@
-      self.version = max + 1
+    audited_changes[attr_name] = if changes.is_a?(Array)
+      changes.map do |value|
+        value&.first(max_value / 2)
```
Added it

heretge commented:
@NoahFisher These changes are directed to get rid of persistent database connection, which is required by any ActiveRecord model, `Audit` model in this case. So, I replaced it with an AR independent entity, removing all the stuff related to versioning and other AR-based features of the original gem. Also, I added monkey patches for `Audit` from our webapp, so they're all in one place.
I agree that some other gem would be a better fit now, but initially I thought it's the way to go to not change too much the webapp integration with it.
